datacube/drivers/postgis/sql.py

```diff
@@ -13,10 +13,11 @@
 from sqlalchemy.sql.expression import Executable, ClauseElement
 from sqlalchemy.sql.functions import GenericFunction
 
-SCHEMA_NAME = 'agdc'
+SCHEMA_NAME = 'odc'
```

Comment:
does this impact user management? will user created from cli on postgis db end up having access to this.

Reply:
Good point, I didn't check that.  I'll have a look.

(I haven't changed the names of the default users yet either - they are still prefixed `agdc-`.  That's less urgent though.)

Reply:
Yes, should work fine as schema_name is already parametised at `datacube/drivers/postgis/_core.py:125-140`